filebeat/.gitattributes

```diff
@@ -1 +1,5 @@
 CHANGELOG.md  merge=union
+
+# Keep these file types as CRLF (Windows).
+*.bat    text eol=crlf
```

Comment:
Yes, it should be. I guess I had assumed that I was in the top-level directory when I made the edit because I expected there to be only one `.gitattributes` for the repo. I will open a PR to correct this and remove the extra `.gitattribute` files.